pkg/telemetry/prometheus/node_linux.go

```diff
@@ -36,6 +37,16 @@ func getCPUStats() (cpuLoad float32, numCPUs uint32, err error) {
 	return
 }
 
+func getMemoryStats() (memoryLoad float32, err error) {
+	memInfo, err := memory.Get()
+	if err != nil {
+		return
+	}
+
+	memoryLoad = float32(memInfo.Used) / float32(memInfo.Total)
```

Comment:
Using `memoryLoad` as `used / total` for a couple of reasons. Another option was to have something like `available / total` and a stat like `memoryRoom`.
1. Looking at this code, `Used` seems to be proper indicator of how much is used. In fact, IIRC, I even got a compile error when using `Available`. Did not dig into why. Anyhow, `Used` looks like the actual used.
2. In some cases like MacOS, the call failed because it could execute the command (maybe requires the process to be run as `root`). In the case of failure, load will report a 0 which has easier semantics to use in capacity manager and rest of the code rather than available showing 100% all the time because the command could not execute.